webapp/templates/jury/partials/verify_form.html.twig

```diff
@@ -3,13 +3,15 @@
 
     {# Display verification data: verified, by whom, and comment #}
     <p>
-        {{ label }}: <strong>{% if judging.verified %}yes{% else %}no{% endif %}</strong>
-        {%- if judging.verified and judging.juryMember is not empty -%}
-            , by {{ judging.juryMember }}
+        {% if judging.verified %}
+            Judging verified
+            {%- if judging.juryMember is not empty -%}
+                , by <code>{{ judging.juryMember }}</code>
```

Comment:
Spelling nitpick (and was already there)
```suggestion
                 by <code>{{ judging.juryMember }}</code>
```

Reply:
Sorry, merged too early, sent a separate change: https://github.com/DOMjudge/domjudge/pull/2671